pandas/plotting/_core.py

```diff
@@ -2704,18 +2704,56 @@ def __call__(self, x=None, y=None, kind='line', ax=None,
 
     def line(self, x=None, y=None, **kwds):
         """
-        Line plot
+        Plot `DataFrame` columns as lines.
+
+        This function is useful to plot lines using `DataFrame`'s values
+        as coordinates.
+        It allows to specify which columns to use for the axes.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : int, str, list of int or list of str, optional
```

Comment:
I think x has to be an int or str. Lists are not allowed for x, just y.

Reply:
We tested on Jupyter and these seem to be allowed. However, the result makes not much sense. I'm changing this, anyway.